extension/src/processManager.ts

```diff
@@ -55,17 +55,6 @@ export class ProcessManager extends Disposable {
     return this.runQueued()
   }
 
-  public async forceRunQueued(): Promise<void> {
```

Comment:
[F] Only place this was called was a test. I was in this class in the first place to try and stop the queue from being run after it had been disposed of.